### ISSUE TITLE
Cocoapod Error 

### ISSUE THREAD

**Issue by sarankumar291** [rs/SDWebImage#2145]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / tvos
 Platform Version        | e.g. 8.0
 SDWebImage Version      | e.g. 3.7.6
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 7.3
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

[!] Unable to satisfy the following requirements:

- `SDWebImage` required by `Podfile`
- `SDWebImage (!= 3.7.2, ~> 3.7)` required by `MWPhotoBrowser (2.1.2)`

Specs satisfying the `SDWebImage, SDWebImage (!= 3.7.2, ~> 3.7)` dependency were found, but they required a higher minimum deployment target.

**Comment by dreampiggy** [comment]
This should match the version 3.7.6. But if CocoaPods say "they required a higher minimum deployment target.", check your minimum target version(iOS 5 & tvOS 9)

https://github.com/rs/SDWebImage/blob/3.7.6/SDWebImage.podspec
